.ado/pipelines/templates/stages-full-release.yaml

```diff
@@ -305,11 +305,11 @@ stages:
       parameters:
         stageName: 'deployChaosExperiments'
         customPrefix: '$(prefix)$(customSuffix)'
-        testDurationSeconds: '$(chaosExperimentDurationSeconds)' # Chaos test duration for each experiment
+        testDurationSeconds: '$(chaosExperimentDurationSeconds)'
```

Comment:
I realize now that this isn't the most accurate name - we're not running chaos "tests". How about `experimentDurationSeconds` to indicate that **each** experiment will have this duration (regardless of running in parallel or sequentially)? Which is basically what the removed comment says.

Reply:
yep will change

Reply:
done